Clicker/ViewControllers/GroupControlsViewController.swift

```diff
@@ -136,38 +124,12 @@ class GroupControlsViewController: UIViewController {
     }
 
     private func setupConstraints() {
-        infoView.snp.makeConstraints { make in
-            make.top.equalTo(view.safeAreaLayoutGuide.snp.top).offset(infoViewTopPadding)
-            make.leading.trailing.equalToSuperview()
-            make.height.equalTo(infoViewHeight)
-        }
-
-        attendanceLabel.snp.makeConstraints { make in
-            make.top.equalTo(infoView.snp.bottom).offset(attendanceLabelTopPadding)
-            make.leading.trailing.equalToSuperview().inset(attendanceLabelHorizontalPadding)
-        }
-
         collectionView.snp.makeConstraints { make in
-            make.leading.trailing.equalTo(attendanceLabel)
-            make.top.equalTo(attendanceLabel.snp.bottom).offset(collectionViewTopPadding)
-            make.height.equalTo(collectionViewHeight)
-        }
-
-        exportButton.snp.makeConstraints { make in
-            make.width.equalToSuperview().multipliedBy(exportButtonWidthScaleFactor)
-            make.height.equalTo(exportButtonHeight)
-            make.centerX.equalToSuperview()
-            make.top.equalTo(collectionView.snp.bottom).offset(exportButtonTopPadding)
+            make.leading.trailing.equalToSuperview()
+            make.top.equalToSuperview().offset(collectionViewTopPadding)
+            make.bottom.equalToSuperview()
```

Comment:
can you combine all the equalToSuperview calls to be
`make.leading.trailing.bottom.equalToSuperview()`